docs/src/integrations/source/troubleshooting.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Troubleshooting"
+description: In this page you'll learn how to troubleshoot problems related to access rights when cloning repositories.
```

Comment:
```suggestion
description: Learn how to troubleshoot problems related to access rights when cloning repositories.
```

Because the text appears outside the context of the page.